crates/cxx-qt-lib/src/types/qdate.cpp

```diff
@@ -7,6 +7,21 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 #include "cxx-qt-lib/include/qdate.h"
 
+#include "assertion_utils.h"
+
+#include <cstdint>
+
+// QDate has a single 64-Bit integer member
+// https://codebrowser.dev/qt5/qtbase/src/corelib/time/qdatetime.h.html#QDate::jd
+//
+// https://codebrowser.dev/qt6/qtbase/src/corelib/time/qdatetime.h.html#QDate::jd
+assert_alignment_and_size(QDate, alignof(std::int64_t), sizeof(std::int64_t))
+
+  static_assert(std::is_trivially_copy_assignable<QDate>::value);
```

Comment:
why is this indented?

Reply:
ah, forgot the semicolon at the end of the previous line.
That confuses Clang-format.

Fixed now